install_config/configuring_local.adoc

```diff
@@ -137,7 +156,7 @@ To allow the provisioner pod to delete content on local volumes created by any p
 $ oc create -f https://raw.githubusercontent.com/openshift/origin/master/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml
```

Comment:
using the template: https://raw.githubusercontent.com/openshift/origin/release-3.10/examples/storage-examples/local-examples/local-storage-provisioner-template.yaml